modules/kubernetes/aks-core/k8s-cluser-role-binding-list-ns.tf

```diff
@@ -29,7 +30,8 @@ resource "kubernetes_cluster_role_binding" "edit_list_ns" {
     name = "${each.value.name}-edit-listns"
 
     labels = {
-      "aadGroup" = var.aad_groups.edit[each.key].name
+      "aadGroup"          = var.aad_groups.edit[each.key].name
```

Comment:
Is there a reason this is different to the others?

Reply:
What's different? These rules are cluster-role-binding and not role-bindings.

Reply:
Nice catch, not to my knowladge. I updated it to match now.